StayGo.Domain/Service/IBarService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace StayGo.Domain.Service
+{
+    public interface IBarService<T>
```

Comment:
Why this interface is generic ? 
you are always going to use this service with entity Bar. 
This service is dependent on entity Bar. Each change in Bar will make this service change.